pkg/client_data/lib/page_data.dart

```diff
@@ -33,15 +33,17 @@ class PageData {
 }
 
 /// The server-provided data about the current package.
-@JsonSerializable()
+@JsonSerializable(includeIfNull: false)
 class PkgData {
   final String package;
   final String version;
+  final String publisherId;
```

Comment:
How about giving this a documentation comment that says:
```dart
/// PublisherId of publisher that owns this package, `null` if the package isn't owned by a publisher.
```

If something can be `null`, it's always nice to explain when this is the case.